gossip/src/ping_pong.rs

```diff
@@ -16,6 +16,8 @@ use {
     },
 };
 
+const PING_PONG_HASH_PREFIX: &[u8] = "PING_PONG_DOMAIN".as_bytes();
```

Comment:
Should we use a common prefix namespace like "org.solana.PING_PONG"?

Reply:
i think we should at least include "solana"

Reply:
Added `SOLANA` as prefix.